recipe/meta.yaml

```
@@ -44,17 +44,19 @@ test:
   requires:
     - boto3 >=1.17.46
     - composeml >=0.3.0
-    - fastparquet==0.5.0
-    - graphviz !=2.47.2
-    - moto >=1.3.15
+    - graphviz >=0.8.4
+    - moto[all] >=1.3.15
     - pip >=21.3.1
-    - pympler >=0.8
-    - pytest >=5.2.*
-    - python-graphviz >=0.8.4
-    - smart_open >=5.0.*
-    - urllib3 >=1.26.5
     - pyarrow >=3.0.0
+    - pympler >=0.8
+    - pytest >=5.2.0
+    - pytest-cov >=2.6.1
     - pytest-xdist >=1.26.1
+    - pytest >=5.2.0
+    - smart-open >=5.0.0
```

```suggestion
    - smart_open >=5.0.0
```
https://anaconda.org/conda-forge/smart_open